src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -903,11 +903,30 @@ static String getPropertyOnlyName(String name, Logger logger) {
         return null;
     }
 
+    private final static String[] systemPropertiesToLog = new String[] { "java.specification.vendor",
+            "java.specification.version", "java.class.path", "java.class.version", "java.runtime.name",
+            "java.runtime.version", "java.vendor", "java.version", "java.vm.name", "java.vm.vendor", "java.vm.version",
+            "java.vm.specification.vendor", "java.vm.specification.version", "os.name", "os.version", "os.arch" };
```

Comment:
I'm not very sure if these System properties are "required" to be logged by the driver. Reason being, they are not owned/provided by the driver, if anyone wants to log, they could do themselves as they're readily available in JVM.

Reply:
The intent is to gather information that is sometimes left out during support but that affects driver functionality. I've encountered issues that can be traced to differences around these settings.